Documentation/install/guides/kops.rst

```diff
@@ -0,0 +1,31 @@
+**********************************
+Kubernetes Kops Installation Guide
+**********************************
+
+As of kops 1.9 release, Cilium is supported as CNI plugin for kops-deployed clusters.
+
+Cilium needs newer kernel version than default kops images, so you need to supply an ami which will be new enough for Cilium to run on it.
+
+CoreOS images have new enough kernels for Cilium to run on them and are tested by kops developers, which makes them a perfect candidate.
+
+The latest stable CoreOS AMI can be found using aws cli:
+
+.. code:: bash
+
+    $ aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
```

Comment:
Can you provide the output the user is expected to see?

Reply:
Done